Source/FilteringFunctions/arm_biquad_cascade_df2T_f64.c

```diff
@@ -29,415 +29,574 @@
-                   The state array has a total length of <code>2*numStages</code> values.
-                   The state variables are updated after each block of data is processed; the coefficients are untouched.
-  @par
-                   The CMSIS library contains Biquad filters in both Direct Form I and transposed Direct Form II.
-                   The advantage of the Direct Form I structure is that it is numerically more robust for fixed-point data types.
-                   That is why the Direct Form I structure supports Q15 and Q31 data types.
-                   The transposed Direct Form II structure, on the other hand, requires a wide dynamic range for the state variables <code>d1</code> and <code>d2</code>.
-                   Because of this, the CMSIS library only has a floating-point version of the Direct Form II Biquad.
-                   The advantage of the Direct Form II Biquad is that it requires half the number of state variables, 2 rather than 4, per Biquad stage.
-
-  @par           Instance Structure
-                   The coefficients and state variables for a filter are stored together in an instance data structure.
-                   A separate instance structure must be defined for each filter.
-                   Coefficient arrays may be shared among several instances while state variable arrays cannot be shared.
-
-  @par           Init Functions
-                   There is also an associated initialization function.
-                   The initialization function performs following operations:
-                   - Sets the values of the internal structure fields.
-                   - Zeros out the values in the state buffer.
-                   To do this manually without calling the init function, assign the follow subfields of the instance structure:
-                   numStages, pCoeffs, pState. Also set all of the values in pState to zero.
-  @par
-                   Use of the initialization function is optional.
-                   However, if the initialization function is used, then the instance structure cannot be placed into a const data section.
-                   To place an instance structure into a const data section, the instance structure must be manually initialized.
-                   Set the values in the state buffer to zeros before static initialization.
-                   For example, to statically initialize the instance structure use
-  <pre>
-      arm_biquad_cascade_df2T_instance_f64 S1 = {numStages, pState, pCoeffs};
-      arm_biquad_cascade_df2T_instance_f32 S1 = {numStages, pState, pCoeffs};
-  </pre>
-                   where <code>numStages</code> is the number of Biquad stages in the filter;
-                   <code>pState</code> is the address of the state buffer.
-                   <code>pCoeffs</code> is the address of the coefficient buffer;
-*/
+ @defgroup BiquadCascadeDF2T Biquad Cascade IIR Filters Using a Direct Form II Transposed Structure
```

Comment:
Doxygen generation may no more be right here since you removed the indentations.
Keep formatting of Doxygen comments as it was because it is very easy to break things here if you don't regenerate the Doxygen to test.